sdk/communication/azure-communication-chat/tests/test_chat_thread_client_async.py

```diff
@@ -735,3 +679,27 @@ async def mock_send(*_, **__):
         items.append(item)
 
     assert len(items) == 1
+
+@pytest.mark.asyncio
+async def test_get_properties():
+    thread_id = "19:bcaebfba0d314c2aa3e920d38fa3df08@thread.v2"
+    raised = False
+
+    async def mock_send(*_, **__):
+        return mock_response(status_code=200, json_payload={
```

Comment:
Why do we need to use mock responses here? Is there some reason the recording wont work?